README.md

```diff
@@ -40,6 +40,13 @@ To build project please execute the following command:
 ./gradlew clean build
 ```
 
+### Gradle Upgrades
+It is important to run the following command and commit the updated `gradle-wrapper.jar` to the repository for proper gradle upgrades
+
+```bash
+# this sample is for v4.9 replace with correct upgrade version
+./gradlew wrapper --gradle-version 4.9 --distribution-type all
+```
 ### Running
```

Comment:
Just wondered whether we want to use confluence to capture this piece of information instead

https://tools.hmcts.net/confluence/pages/viewpage.action?pageId=397246645&__ncforminfo=vkTo1R57_g5eagNxJXGcP1CjvD8eNe5hst1g9NEoLQTuCINpu0m807BsnF3yeb4s1FLuPt1jSpf8nwg82JdOFY8YIHJt6YJr

Reply:
Thanks for the suggestion. Added a simple page for this https://tools.hmcts.net/confluence/display/RCCD/How+to%3A+Upgrade+Gradle+Version